cmd/coreinstance/create_core_instance_operator.go

```diff
@@ -212,6 +212,20 @@ func newCmdCreateCoreInstanceOperator(config *cfg.Config, testClientSet kubernet
 				return err
 			}
 
+			metadata, err := getCoreInstanceMetadata(k8sClient)
+			if err != nil {
+				return err
+			}
+
+			updateCoreInstance := cloud.UpdateCoreInstance{
```

Comment:
Version and metadata are fields accepted by createCoreInstance, why are we using an update?

Reply:
k8s client needs to be instantiated beforehand and one of the dependencies of the client is the core instance already created before. thats why is later. I can change it if needed.

Reply:
@gabrielbussolo I mean, you are already invoking CreateCoreInstance, the CreateCoreInstanceParams also have a metadata and version fields. Why using a subsequent update?

Reply:
just how about the code was organized, we were using some stuffs for labels on the k8s client after creating the core instance, i changed it